mppm/virtual-players/virtual-players-focus.md

```diff
@@ -12,7 +12,7 @@ To switch between virtual player views in the Multiplayer Play Mode window, use
 ## Focus on a virtual player in the Multiplayer Play Mode window {#focus-play-mode}
 
 1. Open the Multiplayer Play Mode window.
-2. In the Virtual Players section.
+2. Navigate to the Virtual Players section.
 3. Open a player's **More** (**⋮**) menu.
```

Comment:
Are steps 3 and 4 accurate? I couldn't find the More menu or any Focus on Player option in a dropdown.

Reply:
this is accurate to the 1.0.0 release, yes :)